File: 二分查找/153.寻找旋转排序数组中的最小值.java
class Solution {
    public int findMin(int[] nums) {
        int left = -1,right = nums.length-1;
        while(left+1<right){
            int mid = (left+right)/2;
            if(nums[mid]<nums[nums.length-1]){
                right=mid;
            }else{
                left=mid;
            }
        }
        return nums[right];
    }
}

File: 二分查找/153.寻找旋转排序数组中的最小值.go
func findMin(nums []int) int {
	n := len(nums)
	left, right := -1, n-1
	for left+1 < right {
		mid := (left + right) / 2
		if nums[mid] < nums[n-1] {
			right = mid
		} else {
			left = mid
		}
	}
	return nums[right]
}